package.json

```diff
@@ -5,7 +5,7 @@
   "license": "Apache-2.0",
   "homepage": "https://github.com/OceanProtocolEnterprise",
   "scripts": {
-    "start": "npm run pregenerate && next dev -p 8000",
+    "start": "npm run pregenerate && next dev -p 8002",
```

Comment:
I am using port 8008 because I saw it in some tutorial, but I can't find where. You don't have to change it. It is just a suggestion.

Reply:
this is because aquarius is on 8000 when you run with barge and publish with cli

Reply:
ok, i will put 8008